### ISSUE TITLE
How is the desktop font updated?

### ISSUE THREAD

**Issue by callegar** [Templarian/MaterialDesign#6837]
### What is your question?

I see that the webfont is tagged with a version number and has tagged stuff to download as releases. This is not the case for the desktop font that only indicates "sync".  The last update to the desktop font repo is also rather old in comparison to the commits for the webfont. Are releases eventually planned also for the desktop font? Is the desktop font somehow unnecessary (should one use the webfont ttf also for the desktop?). Asking because I use the font with LaTeX (in fact LuaLaTeX).

**Comment by Templarian** [comment]
@callegar That's a bug, it looks like the GitHub Action isn't building the font file and copying it to that repo for some reason. I'll need to look into this.

We don't really monitor that repo so didn't notice.

**Comment by callegar** [comment]
A couple of questions:

- Is the action the one described in `[cd-font-build.yml](https://github.com/Templarian/MaterialDesign/blob/master/.github/workflows/cd-font-build.yml)` ?
- Is the desktop font actually the very same as the `webfont` ttf with just a font name change to avoid conflicts?

**Comment by Templarian** [comment]
@callegar Yes that is exactly how it's configured. It also was meant to be more up to date to encourage designers to submit icons. I have a feeling it's failing due to a token I let expired as the script should still work.

**Comment by lacheteau-s** [comment]
Hi, any news on this issue? I noticed a bunch of icons aren't available from the desktop font compared to what's on the website due to the gap in versions.

**Comment by callegar** [comment]
Also wonder if the font build script could be updated to have the actual font version (e.g., 7.2.96) in the font? Otfinfo currently reports 1.0.
